src/main/java/codeu/controller/ActivityfeedServlet.java

```diff
@@ -1,33 +1,90 @@
 package codeu.controller;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import codeu.model.data.Activity;
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+
 /** This class was mostly modeled using the ConversationServlet as a guide */
 public class ActivityfeedServlet extends HttpServlet{
 	
+	/** Store class that gives access to Users. */
+	private UserStore userStore;
+	
+	private ConversationStore conversationStore;
+	
+	private MessageStore messageStore;
+	
 	/**
 	 * Set up state for handling the activity feed. This method is only called when
 	 * running in a server, not when running in a test.
 	 */
 	@Override
 	public void init() throws ServletException {
 		super.init();
+	    setUserStore(UserStore.getInstance());
+	    setConversationStore(ConversationStore.getInstance());
+	    setMessageStore(MessageStore.getInstance());
 	}
 	
+	public void setUserStore(UserStore userStore) {
+		this.userStore = userStore;
+	}
+	
+	public void setConversationStore(ConversationStore conversationStore) {
+		this.conversationStore = conversationStore;
+	}
+	
+	public void setMessageStore(MessageStore messageStore) {
+		this.messageStore = messageStore;
+	}
+	
+	/** This helper method builds the list by going through the lists of all of the
+	 * users, conversations, and messages and adding them to the list of Activities
+	 * to avoid casting issues. It then sorts the list and reverses the order so that
+	 * it will display correctly. */
+	private List<Activity> buildList() {
+		List<Activity> activity = new ArrayList<>();
+		List<User> users = userStore.getAllUsers();
+		List<Conversation> conversations = conversationStore.getAllConversations();
+		List<Message> messages = messageStore.getAllMessages();
+		// adds users individually to avoid casting with generics issue
```

Comment:
You've got the right idea about not leaving dead code in our PRs before we merge with master, but I think I didn't do a good job of explaining the difference between code that is commented out because we are no longer using it (which should be deleted before we submit a PR) verses comments on our code helping the reader follow what is happening. This is an example of a comment we would want to leave here because it helps the reader understand the code.